build.gradle

```diff
@@ -137,6 +137,10 @@ dependencyManagement {
     dependencySet(group: 'org.springframework', version: '5.3.15') {
       entry 'spring-core'
     }
+    // CVE-2020-36518
+    dependencySet(group: 'com.fasterxml.jackson.core', version: '2.13.2') {
+      entry 'jackson-databind'
+    }
```

Comment:
no need,
https://github.com/hmcts/bulk-scan-end-to-end-tests/pull/286/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R170
bumping this to 2.13.0 should be fine